provision-base.sh

```diff
@@ -60,7 +60,8 @@ install_ssmtp()
 	sed -e "/^root=/ s/postmaster/$TOASTER_ADMIN_EMAIL/" \
 		-e "/^mailhub=/ s/=mail/=haraka/" \
 		-e "/^rewriteDomain=/ s/=\$/=$TOASTER_MAIL_DOMAIN/" \
-		"$BASE_MNT/usr/local/etc/ssmtp/ssmtp.conf.sample" \
+                -i '/^#FromLineOverride=YES/ s/#//' \
+                "$BASE_MNT/usr/local/etc/ssmtp/ssmtp.conf.sample" \
 		> "$BASE_MNT/usr/local/etc/ssmtp/ssmtp.conf" || exit
```

Comment:
Did you test this on the CLI?  I doesn't look to me like it's going to do what you want it to. Also, you appear to have `expandtab` set in vim still. ;-)